index.html

```diff
@@ -42,3 +42,110 @@ <h1 class="hero-title">
         
 </body>
 </html>
+
+<!-- My Recent Work Section -->
+<!-- work section -->
+<section id="recent_work" class="center-content">
```

Comment:
Great job @187jjay187 🤝

- [ ] Kindly make sure the main cards container of the works section has a display grid.
- [ ] Also create individual card sections and make sure each card has a display flex.